docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -138,6 +138,32 @@ $ tsh db ls
 
 </ScopedBlock>
 
+<Admonition type="note" title="Note">
+  You can override the database name by applying the `teleport.dev/database_name` AWS tag to the resource. The value of the tag will be used as the database name.
+</Admonition>
+
+Log in to a particular database using the `tsh db login` command:
+
+```code
+$ tsh db login my-redshift
+```
+
+<Admonition type="tip" title="Tip">
+  You can be logged into multiple databases simultaneously.
+</Admonition>
+
+You can optionally specify the database name and the user to use by default
+when connecting to the database instance:
+
+```code
+$ tsh db login --db-user=awsuser --db-name=dev my-redshift
+```
+
+<Admonition type="note" title="Database users must exist">
+  Teleport does not currently use the auto-create option when generating
+  tokens for Redshift databases. Users must exist in the database.
+</Admonition>
```

Comment:
Why this section was added  ? 

Recently we have added support for `--db-user=awsuser --db-name=dev` flags in `tsh db connect` and `tsh db proxy` so calling 
```
tsh db login --db-user=awsuser --db-name=dev my-redshift
tsh db connect my-redshift
```
before connecting to db is not longer needed. 
A user can just do this in one command: 
```
tsh db connect --db-user=awsuser --db-name=dev my-redshift
```

Reply:
Bad merge; I'll fix it in a moment, thanks for spotting.